_drafts/2024-07-25-testing-raise.md

```diff
@@ -0,0 +1,506 @@
+---
+title: "Raise Your Tests: Testing Arrow Raise"
+date: 2024-04-10
+header:
+    image: "https://res.cloudinary.com/dkoypjlgr/image/upload/f_auto,q_auto:good,c_auto,w_1200,h_300,g_auto,fl_progressive/v1715952116/blog_cover_large_phe6ch.jpg"
+tags: [kotlin, arrow]
+excerpt: ""
+toc: true
+toc_label: "In this article"
+---
+
+_By [Riccardo Cardin](https://github.com/rcardin)_
+
+At RockTheJvm, we deeply understand the power of the Kotlin Arrow library and the Raise DSL, and we've previously shared our insights in our article on [Functional Error Handling in Kotlin, Part 3: The Raise DSL](https://blog.rockthejvm.com/functional-error-handling-in-kotlin-part-3/). Now, we're ready to delve into the crucial topic of testing applications that use the Raise DSL. To fully grasp the concepts we'll be discussing, it's essential to have a solid understanding of Kotlin, Arrow library, and the Raise DSL. We recommend revisiting our previous posts or resources if you need to refresh your knowledge. Get ready to explore the testing world of Arrow Raise with confidence and understanding.
```

Comment:
Avoid "delve" because people have ChatGPT anxiety 😄

Reply:
I didn't know about the "delve" fact. Absurd.